lib/mavlink/v1.0

```diff
@@ -1 +1 @@
-Subproject commit 6251d3f235d653e88eec8f599787adfbd361f358
+Subproject commit fc2580177a4e2422d4f7f38b5c59a100064b1fe8
```

Comment:
@superjax So there's a script that's part of the mavlink repo ([update_c_library.sh](https://github.com/BYU-MAGICC/mavlink/blob/master/scripts/update_c_library.sh)) for autogenerating the headers and pushing them to the `mavlink_c_library` repo. I think for consistency, and to avoid problems like missing files, it'd be better to use that rather than adding in the new files manually. I've done that and updated this submodule reference to point to the new commit.